docs/sources/tempo/operations/tempo_cli.md

```diff
@@ -306,8 +306,9 @@ tempo-cli migrate tenant --source-config source.yaml --config-file dest.yaml my-
 
 ## Analyse block
 Analyses a block and outputs a summary of the block's generic attributes.
+<!-- Commenting out until vParquet3 is available.
```

Comment:
same comment here. these are not in the 2.2 branch so we don't need to backport redactions.

https://grafana.com/docs/tempo/latest/operations/tempo_cli/

Reply:
so I should just undo the comments?

Reply:
```suggestion

```